init.ps1

```diff
@@ -0,0 +1,152 @@
+[CmdletBinding()]
+Param(
+	[Parameter(Mandatory=$false)]
+	[switch]$build,
+
+	[Parameter(Mandatory=$false)]
+	[switch]$run,
+
+	[Parameter(Mandatory=$false)]
+	[switch]$docs,
+
+	[Parameter(Mandatory=$false)]
+	[switch]$setup_dev_env,
+
+	[Parameter(Mandatory=$false)]
+	[switch]$verify,
+
+	[Parameter(Mandatory=$false)]
+	[switch]$test,
+
+	[Parameter(Mandatory=$false)]
+	[switch]$test_unit,
+
+	[Parameter(Mandatory=$false)]
+	[switch]$test_e2e,
+
+	[Parameter(Mandatory=$false)]
+	[switch]$docker,
+
+	[Parameter(Mandatory=$false)]
+	[switch]$olympus_docker,
+
+	[Parameter(Mandatory=$false)]
+	[switch]$proxy_docker,
+
+	[Parameter(Mandatory=$false)]
+	[switch]$bench,
+
+	[Parameter(Mandatory=$false)]
+	[switch]$alldeps,
+
+	[Parameter(Mandatory=$false)]
+	[switch]$dev,
+
+	[Parameter(Mandatory=$false)]
+	[switch]$down
+)
+function execScript($name) {
+	$scriptsDir = "$(join-path scripts ps)"
+	& "$(Join-Path $scriptsDir $name)"
+}
+
+if ($build.IsPresent) {
+	try {
+		Push-Location $(Join-Path cmd proxy)
+		& buffalo build
+	}
+	finally {
+		Pop-Location
+	}
+	
+	try {
+		Push-Location $(Join-Path cmd olympus) 
+		& buffalo build
+	}
+	finally {
+		Pop-Location
+	}
+}
+
+if ($run.IsPresent) {
+	Set-Location $(Join-Path cmd proxy)
+	& buffalo dev
+}
+
+if ($docs.IsPresent) {
+	Set-Location docs 
+	& hugo
+}
+
+if ($setup_dev_env.IsPresent) {
```

Comment:
can we place setupdevenv before build?
and dev/alldeps before tests?
so you can say `init -setupdevenv build alldeps test-unit` and you have a killer combo